cairo/tests/test_serde.py

```diff
@@ -258,19 +269,26 @@ def test_type(
             MessageCallOutput,
             Union[Bytes, LegacyTransaction],
             Union[Bytes, Receipt],
+            Union[Bytes, Withdrawal],
             Optional[Union[Bytes, LegacyTransaction]],
             Optional[Union[Bytes, Receipt]],
+            Optional[Union[Bytes, Withdrawal]],
+            Mapping[Bytes, Optional[Union[Bytes, LegacyTransaction]]],
+            Mapping[Bytes, Optional[Union[Bytes, Receipt]]],
+            Mapping[Bytes, Optional[Union[Bytes, Withdrawal]]],
             Tuple[Union[Bytes, LegacyTransaction], ...],
             Block,
             List[Block],
             BlockChain,
             Trie[Bytes, Optional[Union[Bytes, LegacyTransaction]]],
             Trie[Bytes, Optional[Union[Bytes, Receipt]]],
+            Trie[Bytes, Optional[Union[Bytes, Withdrawal]]],
         ],
     ):
         assume(no_empty_sequence(b))
         assume(single_evm_parent(b))
         type_ = get_type(b)
+        b = remove_none_values(b)
```

Comment:
Why adding this is now?
What changed that would requires to remove the none_values from b?

Reply:
we added Mappings[T, Optional[U]] types